app/models/restaurant.rb

```diff
@@ -0,0 +1,39 @@
+# == Schema Information
+#
+# Table name: restaurants
+#
+#  id         :integer          not null, primary key
+#  name       :string
+#  rating     :decimal(, )      default(0.0)
+#  tenbis     :boolean
+#  address    :string
+#  price      :decimal(, )
+#  maxDevTime :decimal(, )
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  cuisine_id :integer
+#
+
+class Restaurant < ApplicationRecord
+  has_many :reviews , dependent: :destroy
+  has_many :users , :through => :reviews
+  belongs_to :cuisine
+
+
+  #validates :name , :address ,presence:true,allow_nil:false, allow_blank: true
+  validates :name, :presence => true, :allow_nil => false, :allow_blank => false
```

Comment:
if you validate for presence, do you really need to tell it to `:allow_nil => false, :allow_blank => false`?

Reply:
yes,
if i won't validate it , the user could submit ""